e2e/runner/run_cypress_local.js

```diff
@@ -0,0 +1,135 @@
+const CypressBackend = require("./cypress-runner-backend");
```

Comment:
Wdyt about converting `e2e/runner/run_cypress_local.js` to the `.ts` file and run it via `tsx e2e/runner/run_cypress_local.ts`. From what I see it requires a minimum set of changes (a few types).

It will heavily simplify my work on https://www.notion.so/metabase/Ensure-the-usage-of-the-latest-Embedding-SDK-code-in-documentation-snippets-code-snippets-etc-18969354c90180d5a6a5d9237bb46431?pvs=4#18a69354c9018079a65acba9f3a45392 

Because I have to adapt my POC for this PR, I write the `.ts` code. And without the conversion, I have to `spawn` a process with my file, or convert all my files to `.js`, what I'd like to avoid. 
If this file is converted, I can just import my logic here and run it for a new test suite type

Reply:
Absolutely. I wanted to add zero dependencies, but I forgot we had `tsx` in the repo already 👍